components/camel-mongodb/src/main/java/org/apache/camel/component/mongodb/MongoDbEndpoint.java

```diff
@@ -68,6 +69,13 @@
     @UriPath(description = "Sets the connection bean reference used to lookup a client for connecting to a database.")
     @Metadata(required = true)
     private String connectionBean;
+
+    @UriParam
+    private String username;
+    @UriParam
```

Comment:
(label = security, secret = true)

to have username and password as security options and also as secret so they are not leaked in logs etc

Reply:
thanks!